.scrutinizer.yml

```diff
@@ -12,7 +16,11 @@ build:
     tests:
         override:
             -
-                command: 'make coverage'
+                command: 'make test-coverage -B'
```

Comment:
what does this `-B` do?

Reply:
If a folder exists with the same name as the command sometimes Make returns that it's already up to date. `-B` forces a build` with the new command name its unnecessary